packages/date-picker/src/panel/date.vue

```diff
@@ -127,6 +127,13 @@
           @click="changeToNow">
           {{ t('el.datepicker.now') }}
         </el-button>
+        <el-button
+          size="mini"
+          type="warning"
```

Comment:
Styling should be consistent with time-panel's cancel

Reply:
Time-panel's cancel uses native `button` element. Should I use `el-button` with `type="text"`? `plain` doesn't work for text type though. It'll look like this.
![screen shot 2018-03-06 at 1 22 35 pm](https://user-images.githubusercontent.com/11173080/37018084-7701e280-2141-11e8-81c4-c826fc9feac4.png)

Reply:
I think you can simply copy & paste time-panel's button.
May need to copy styles as well. (see `packages/theme-chalk/`)